test/chpldoc/compflags/version/projectVersion.dotEnd.doc.good

```diff
@@ -0,0 +1,11 @@
+error: Invalid version format: 21.0. due to: Cannot end with dot, can end with either number or tag
+var DOCUMENTATION_OPTIONS = {
```

Comment:
It worries me a little to have the .js from a correct version included in the expected output of a failure mode.  I think you can avoid this by having all the error-generating modes run before the correct ones (or by splitting them into separate tests that write to a different output directory).